src/net.rs

```diff
@@ -325,7 +328,9 @@ impl NetworkConnector for HttpConnector {
                 debug!("https scheme");
                 let stream = try!(TcpStream::connect(addr));
                 let mut context = try!(SslContext::new(Sslv23).map_err(lift_ssl_error));
-                self.0.as_ref().map(|cb| context.set_verify(SslVerifyPeer, Some(*cb)));
+                if let Some(ref v) = self.0 {
```

Comment:
Can we use a more descriptive name like `verifier`? It's a small change but it brings clarity.